src/pkgcheck/checks/python.py

```diff
@@ -422,3 +426,196 @@ def feed(self, pkg):
                 if GITHUB_ARCHIVE_RE.match(uri):
                     yield PythonGHDistfileSuffix(f.filename, uri, pkg=pkg)
                     break
+
+
+class PythonHasVersionUsage(results.VersionResult, results.Style):
+    """Package uses has_version inside ``python_check_deps``.
+
+    An ebuild that declares ``python_check_deps`` function, which tests
```

Comment:
Ebuilds which declare the ``python_check_deps`` function (which tests Python implementations
for matching dependencies) should use the special ``python_has_version`` function (instead
of ``has_version``) for enhanced log output and defaults.